nilearn/interfaces/tests/test_bids.py

```diff
@@ -16,9 +18,177 @@
     parse_bids_filename,
     save_glm_to_bids,
 )
+from nilearn.interfaces.bids.query import (
+    _get_metadata_from_bids,
+    _infer_repetition_time_from_dataset,
+    _infer_slice_timing_start_time_from_dataset,
+)
 from nilearn.maskers import NiftiMasker
 
 
+def test_get_metadata_from_bids(tmp_path):
+
```

Comment:
a one-linear to explain what this does ?

Reply:
added some explanations: let me know if that's enough